src/main/java/seedu/address/ui/InfoPanel.java

```diff
@@ -68,17 +68,14 @@
      * Creates a {@code InfoPanel} with the given {@code Person}.
```

Comment:
Perhaps the header comment here could be updated with regards to how `ObservablePerson` is used?

Reply:
Suggestion Taken!